### DISCUSSION TITLE
Simple question on apk tool and permissions

### DISCUSSION THREAD

**Discussion by ChillVibesMushroom** [iBotPeaches/Apktool#2898] [General]
When I look at the Android manifest file I see I can add permissions what if I wanted to add a permission would I just put it there and recompile?

**Comment by iBotPeaches** [comment]
I moved this to discussions as not really a bug.

You could easily decode the application, which leaves the `AndroidManifest.xml` in readable XML. Edit in the permissions you want and rebuild.
